src/components/LoginForm/styles.module.scss

```diff
@@ -0,0 +1,19 @@
+@import 'variables/colors';
+
+.login-form {
+  background-color: $white-main;
+  border-top: 6px solid $blue;
+  display: flex;
```

Comment:
We should have a `column` global class you can use instead of this two properties